public/style.css

```diff
@@ -123,17 +124,28 @@ ul {
 }
 
 .nav {
+  display: none;
   height: 100%;
   width: 100%;
   position: fixed;
   background-color: var(--purple);
   overflow: hidden;
-  max-height: 0;
-  top: 80px;
-  z-index: 20;
+  max-height: 100%;
+  top: 0;
+  z-index: 10;
+  padding-top: 25px;
   transition: max-height 0.5s ease-out;
 }
 
+.nav div {
+  padding: 15px;
+  font-size: 1.7em;
+  color: var(--white);
+  text-align: right;
+  margin-right: 15px;
+  cursor: pointer;
+}
```

Comment:
Nice job implementing the functionalities of the mobile menu👍. Let's try to make the mobile menu look more like the design as we ought to stick to the design in this milestone👍

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|![image](https://user-images.githubusercontent.com/64643995/232521322-3b614c44-f9ef-4154-8b68-26baf103cdc2.png)|![image](https://user-images.githubusercontent.com/64643995/232521401-439ee288-bc80-4ca9-8591-c861b8b2acbc.png)|

Kindly make a comparison and make the necessary changes🥲 Take note of the blur property on the design and the font property of your texts👍.